site/frontend/src/pages/graphs/plots.ts

```diff
@@ -244,7 +244,7 @@ function normalizeData(data: GraphData) {
 
 // Renders the plots data with the given parameters from the `selector`, into a DOM node optionally
 // selected by the `elementSelector` query.
-export function renderPlots(data: GraphData, selector: GraphsSelector, elementSelector: string = "#charts") {
+export function renderPlots(data: GraphData, selector: GraphsSelector, elementSelector: string) {
```

Comment:
random note: I added this default arg mostly because of https://github.com/rust-lang/rustc-perf/blob/2ceca416227133dca8e20664b313a5e22539e855/site/frontend/src/pages/bootstrap/page.vue#L17 but hadn't checked if typechecking detected it as an error.

Reply:
It hopefully shouldn't have anything to do with bootstrap, that's a different function (albeit with the same name). I would rather specify the ID explicitly on a single place in `content.vue` rather than having a magic default there, that's why I changed it back.

Reply:
oh I see, it's the same filename and function but in a different folder 🤦 . 

I also had done it this way first before changing to the default because I hadn't noticed the bootstrap page wasn't calling it ... I prefer that as well, so the id is defined and used in the same file.